fleetctl/fleetctl.go

```diff
@@ -732,6 +803,64 @@ func lazyCreateUnits(args []string) error {
 	return nil
 }
 
+// matchUnitFiles compares two unitFiles
+// Returns true if the units match, false otherwise.
+func matchUnitFiles(a *unit.UnitFile, b *unit.UnitFile) bool {
+	if a.Hash() == b.Hash() {
+		return true
+	}
+
+	return false
+}
```

Comment:
Ok that one was introduced in that commit to make it clear what we are doing, then that code was moved into another unit file so it's shared and used by fleetd too.